src/azure-cli/azure/cli/command_modules/resource/aaz/profile_2020_09_01_hybrid/account/_list_locations.py

```diff
@@ -0,0 +1,168 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "account list-locations",
+)
```

Comment:
It is still `az account list-locations`. Should we rename it to `az subscription list-locations` as requested by https://github.com/Azure/azure-cli/pull/27400#discussion_r1328211046?

Reply:
In order not to break the customer, we will temporarily keep the design of this name. In the subsequent sprint, we will let the `account list-locations` command go through the deprecated process and replace it with `az subscription list-locations` @Jing-song Please be aware of it